frontend/src/components/App/Layout.tsx

```diff
@@ -47,7 +47,7 @@ function ClusterNotFoundPopup() {
       alignItems="center"
     >
       <Box p={0.5}>{t(`Something went wrong with cluster {{ cluster }}`, { cluster })}</Box>
-      <Button variant="contained" size="small" href="/">
+      <Button variant="contained" size="small" href={window.desktopApi ? '#' : '/'}>
```

Comment:
Hmm, can you add the fixes as a different commit? Sorry I didn't ask for this before.

Reply:
but this is a different commit @joaquimrocha